src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final String INJECTOR_INSTANCE = "mate";
+    private static final Injector injector = Injector.getInstance(INJECTOR_INSTANCE);
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> allDrivers = driverService.getAll();
+        req.setAttribute("alldrivers", allDrivers);
+        List<Car> allCars = carService.getAll();
+        req.setAttribute("allcars", allCars);
+        req.getRequestDispatcher("/WEB-INF/views/car/addDriverToCar.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        carService.addDriverToCar(driverService.get(Long.parseLong(req.getParameter("driverId"))),
+                carService.get(Long.parseLong(req.getParameter("carId"))));
```

Comment:
```suggestion
        Driver driver = driverService.get(Long.parseLong(req.getParameter("driverId"));
        Car car = carService.get(Long.parseLong(req.getParameter("carId")))
        carService.addDriverToCar(driver, car);
```

It is really hard to read such code

Reply:
Done